docs/pages/next-steps/additional-resources.md

```diff
@@ -42,4 +42,5 @@ The following resources are useful for learning Expo and some of the projects th
 
 ## JavaScript
 
+- [ES6 by Wes Bos](https://wesbos.com/es6-for-everyone) - ES6 for Everyone! (free)
```

Comment:
```suggestion
- [ES6 by Wes Bos](https://es6.io/) (paid)
```

Seems like the course is paid and the link is now https://es6.io/. Left a suggestion to update the changes.